File: String/Furthest Point From Origin/Furthest_Point_From_Origin.java
//Problem Name: Furthest Point From Origin
//Problem Link: https://leetcode.com/problems/furthest-point-from-origin/

class Solution {
    public int furthestDistanceFromOrigin(String moves) {
        int n = moves.length();
        int cntl = 0, cntr = 0;
        for (int i = 0; i < n; i++) {
            if (moves.charAt(i) == '_') {
                cntl++;
                cntr++;
            } else if (moves.charAt(i) == 'L') {
                cntl++;
                cntr--;
            } else if (moves.charAt(i) == 'R') {
                cntr++;
                cntl--;
            }
        }
        return Math.max(cntl, cntr);
    }
}

File: String/Furthest Point From Origin/Furthest_Point_From_Origin.cpp
//Problem Name: Furthest Point From Origin
//Problem Link: https://leetcode.com/problems/furthest-point-from-origin/

class Solution {
public:
    int furthestDistanceFromOrigin(string moves) {
        int n = moves.size();
        int cntl = 0, cntr = 0;
        for (int i = 0; i < n; i++) {
            if (moves[i] == '_') {
                cntl++;
                cntr++;
            } else if (moves[i] == 'L') {
                cntl++;
                cntr--;
            } else if (moves[i] == 'R') {
                cntr++;
                cntl--;
            }
        }
        return max(cntl, cntr);
    }
};